pkg/route/validation/validation.go

```diff
@@ -212,6 +212,27 @@ func validateRoute(ctx context.Context, route *routev1.Route, checkHostname bool
 		result = append(result, errs...)
 	}
 
+	if errs := ValidatePathWithSpecRoute(route, specPath.Child("path")); len(errs) != 0 {
```

Comment:
See https://github.com/openshift/library-go/blob/master/pkg/route/validation/validation.go#L221-L222 for an idea on how to implement ratcheting, e.g. making sure that this applies only to new routes being created, and not to old routes being updated.